content/lib/pencil-tracer.js

```diff
@@ -946,6 +1014,28 @@
       return results;
     };
 
+    JavaScriptInstrumenter.prototype.substringByLocation = function(loc) {
+      var j, lineNum, ref, ref1, result;
+      result = "";
+      for (lineNum = j = ref = loc.start.line, ref1 = loc.end.line; ref <= ref1 ? j <= ref1 : j >= ref1; lineNum = ref <= ref1 ? ++j : --j) {
```

Comment:
Same comment as above -- make some sort of iterator, to capture what this is doing. I can't understand this as it is.